nepal_compliance/public/js/calendar_theme.js

```
@@ -0,0 +1,71 @@
+const THEMES = {
+    DARK: 'dark',
+    LIGHT: 'light'
+};
+
+function applyCalendarTheme(theme) {
+    if (!theme || typeof theme !== 'string') {
+        theme = THEMES.LIGHT;
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)